test/FastTests/Client/QueriesWithCustomFunctions.cs

```diff
@@ -2132,6 +2133,125 @@ select new
             }
         }
 
+        [Fact]
+        public async Task QueryCmpXchgValue(){
+        
+            using (var store = GetDocumentStore())
+            {
+                await store.Operations.SendAsync(new PutCompareExchangeValueOperation<string>("users/1", "Karmel", 0));
```

Comment:
Shouldn't this be the other way around? The unique is `Karmel` and the value is `users/1`?